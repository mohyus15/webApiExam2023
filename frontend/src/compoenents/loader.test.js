import React from 'react';
import renderer from 'react-test-renderer';
import Loader from './loader.js';
it('renders correctly', () => {
	const snap = renderer.create(<Loader />);
	expect(snap).toMatchSnapshot();
});
