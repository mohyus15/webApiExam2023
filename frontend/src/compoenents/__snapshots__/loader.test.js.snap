// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="spinner-border"
  role="status"
  style={
    Object {
      "display": "block",
      "height": 100,
      "margin": "auto",
      "width": 100,
    }
  }
/>
`;
